examples/fwd_midi.rs

```diff
@@ -3,22 +3,22 @@ extern crate vst;
 
 use vst::api;
 use vst::buffer::{AudioBuffer, SendEventBuffer};
-use vst::event::{Event, MidiEvent};
 use vst::plugin::{CanDo, HostCallback, Info, Plugin};
 
 plugin_main!(MyPlugin); // Important!
 
 #[derive(Default)]
 struct MyPlugin {
     host: HostCallback,
-    events: Vec<MidiEvent>,
+    recv_buffer: SendEventBuffer,
     send_buffer: SendEventBuffer,
 }
 
 impl MyPlugin {
     fn send_midi(&mut self) {
-        self.send_buffer.send_events(&self.events, &mut self.host);
-        self.events.clear();
+        self.send_buffer
+            .send_events(self.recv_buffer.events().events(), &mut self.host);
```

Comment:
You should just use one `SendEventBuffer` here and call `host.process_events(self.events.events())`; no need to copy them into a second buffer.